graphql.go

```diff
@@ -69,6 +69,7 @@ type Schema struct {
 	logger                log.Logger
 	useStringDescriptions bool
 	disableIntrospection  bool
+	prefixRootFunctions   bool
```

Comment:
Why do we need this?

Reply:
Yes, please. I would prefer one logical change per PR. Please, remove the prefixes of root functions from this PR.